CMakeLists.txt

```diff
@@ -13,10 +13,14 @@ set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
 add_definitions(-DIVAN_VERSION="${PROJECT_VERSION}" -DUSE_SDL)
 
 option(BUILD_MAC_APP "Build standalone application for MacOS" OFF)
+option(USE_HOME_FOR_STATE_DIR "Statedir will be /.ivan/ in current user's homedir" OFF)
 
 if(UNIX)
   add_definitions(-DUNIX)
   include(GNUInstallDirs)
+  if(USE_HOME_FOR_STATE_DIR)
```

Comment:
The `LOCAL_STATE_DIR` define below should be enclosed in `if(USE_HOME_FOR_STATE_DIR)`.

Reply:
I'm afraid that this PR is already merged. But you can take a look at the [new one](https://github.com/Attnam/ivan/pull/495).